appcli/commands/task_cli.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Commands for application tasks.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# standard libraries
+import sys
+
+# vendor libraries
+import click
+
+# local libraries
+from appcli.logger import logger
+from appcli.models.configuration import Configuration
+
+# ------------------------------------------------------------------------------
+# CLASSES
+# ------------------------------------------------------------------------------
+
+
+class TaskCli:
+
+    # --------------------------------------------------------------------------
+    # CONSTRUCTOR
+    # --------------------------------------------------------------------------
+
+    def __init__(self, configuration: Configuration):
+        self.cli_configuration: Configuration = configuration
+        self.orchestrator = configuration.orchestrator
+
+        # ----------------------------------------------------------------------
+        # PUBLIC METHODS
+        # ----------------------------------------------------------------------
+
+        @click.group(
+            invoke_without_command=True,
+            help="Commands for application tasks.",
+        )
+        @click.pass_context
+        def task(ctx):
+            if ctx.invoked_subcommand is not None:
+                # subcommand provided
+                return
+
+            click.echo(ctx.get_help())
+
+        @task.command(
+            help="Runs a specified application task.",
+            context_settings=dict(ignore_unknown_options=True),
+        )
+        @click.argument("service_name", required=True, type=click.STRING)
+        @click.argument("extra_args", nargs=-1, type=click.UNPROCESSED)
+        @click.pass_context
+        def run(ctx, service_name, extra_args):
```

Comment:
I'm not sure I see the value of the group.

Can you think of other things which would live here?

As a sysadmin I think I'd rather
```
./appcli task backup-db

# over
./appcli task run backup-db
```

Not fussed, if you think there's likely things we'll add under here then that's fine

Reply:
I think there's some task-specific commands that we may want to implement later, and having the group means that we don't have to change the api for running a task.
For example, listing all tasks that can be run, or list currently-running tasks (if being run as a daemon).